python/build-wheel-manylinux2014.sh

```diff
@@ -35,14 +35,20 @@ echo "java_bin path $java_bin"
 java_home=${java_bin%jre/bin/java}
 export JAVA_HOME="$java_home"
 
-/ray/ci/env/install-bazel.sh
+WORK_DIR=${WORK_DIR:-}
```

Comment:
it gets passed from the CCI config.yml. In case someone wanted to run it via the command mentioned here https://hyperscience.atlassian.net/wiki/spaces/DEVOPS/pages/3379560476/Building+ray#Building-for-linux.
But don't imagine anyone doing that, so will remove it